### ISSUE TITLE
[Designer] Bug 33490110: [Food Order.json]: Keyboard tab focus is not shifted to newly opened dialog after activating “Steak" button on 'FoodOrder.json' card.

### ISSUE THREAD

**Issue by aaagrawal** [microsoft/AdaptiveCards#7622]
### Problem Description

[Bug 33490110](https://microsoft.visualstudio.com/OS/_workitems/edit/33490110): [Food Order.json]: Keyboard tab focus is not shifted to newly opened dialog after activating “Steak" button on 'FoodOrder.json' card.

Test Environment:
Application: Adaptive Cards
OS version: Dev (OS Build 21387.1)
Application Version: 2.7.2105.11001

Repro steps:

1. Launch the Adaptive Card application.

2. Adaptive Card UWP Visualizer screen starts appearing. 

3. Navigate to FoodOrder.Json button by using Tab key and activate it.

4. FoodOrder adaptive card will be appeared on right sight of the pane.

5. Navigate to FoodOrder card by using Tab key.

6. Navigate to "Steak" button and activate it.

7. Observe the keyboard issue.

Actual Result:
Keyboard tab focus is not shifted to newly opened dialog after pressing “Steak" button.

Expected Result:
Keyboard tab focus should be shifted to the next interactive element i.e., "Rare" radio button present under newly opened dialog after pressing “Steak"  button.

User Experience:
Keyboard Users might get affected as keyboard focus is moving to the next button "Chicken" when “Steak" button is selected. Due to this behavior, user are getting impacted as they have to press more tabs to reach back the newly opened dialog.

Note: 
Same issue is also repro with "Steak", "Chiken" and "Tofu buttons on Food order with Validation card.
Same issue is also repro with "Chicken" and "Tofu" buttons on 'FoodOrder.json' card.

### Card JSON

```JSON
Designer Bug. Repro on UWP Adaptive Card visualizer
```

**Comment by vagpt** [comment]
Hi @anna-dingler,

Could you please provide us update on this issue? As issue is still repro and after activating the steak control focus should be land on the activated control.
